flipt-evaluation/src/parser/mod.rs

```diff
@@ -1,34 +1,71 @@
+use reqwest::header::HeaderMap;
+use serde::Deserialize;
+
 use crate::error::Error;
 use crate::models::source;
 
 pub trait Parser {
     fn parse(&self, namespace: &str) -> Result<source::Document, Error>;
 }
 
+#[derive(Deserialize, Clone)]
+pub struct Authentication {
```

Comment:
could this be an enum since its an either or kind of thing and not both?

Reply:
@markphelps Good suggestion. I made the change to use an `enum` instead. Thank you!